text/0124-documents-api.md

```diff
@@ -121,6 +122,12 @@ Gives the `limit` parameter used for the query.
 
 > See [3.1.1.2.2. `limit`](#31122-limit) section.
 
+###### 3.1.1.3.3. `total`
+
+- Type: Integer
+- Required: True
+
+Gives the total number of documents that can be browsed in the related index.
```

Comment:
Same question as in the other total: actual or estimated?

Reply:
https://github.com/meilisearch/specifications/pull/138#discussion_r852920472